src/seedu/addressbook/parser/Parser.java

```diff
@@ -57,50 +43,17 @@
     public static final Pattern BASIC_COMMAND_FORMAT = Pattern.compile("(?<commandWord>\\S+)(?<arguments>.*)");
 
     /**
-     * Parses user input into command for execution.
-     *
-     * @param userInput full user input string
-     * @return the command based on the user input
+     * Extracts the new person's tags from the add command's tag arguments string.
```

Comment:
All non trivial methods should have preferably have javadoc format header comments.